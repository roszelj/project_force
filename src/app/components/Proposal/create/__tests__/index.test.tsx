import * as React from 'react';
import { render } from '@testing-library/react';
import { createRenderer } from 'react-test-renderer/shallow';

import { ProposalForm } from '..';

const shallowRenderer = createRenderer();

describe('<ProposalForm />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<ProposalForm />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
