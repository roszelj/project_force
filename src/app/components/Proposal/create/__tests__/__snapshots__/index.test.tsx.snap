// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProposalForm  /> should match snapshot 1`] = `
<div
  class=""
/>
`;
